src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,54 @@
+.container {
+  display: flex;
+  height: 100vh;
+  background-color: #fffbfb;
```

Comment:
you created variables but not for all colors?)
```suggestion
  background-color: $someColor;
```

Reply:
yep, because i use it only once) there is a part in checklist about this